docs/source/cli_help/traverse.rst

```diff
@@ -65,6 +65,23 @@ It will pick up the walk from the current branch (unless ``--start-from=`` or ``
 Unlike with e.g. ``git rebase``, there is no special ``--continue`` flag, as ``traverse`` is stateless
 (doesn't keep a state of its own like ``git rebase`` does in ``.git/rebase-apply/``).
 
+The rebase and push behaviour of ``traverse`` can also be customized for each branch separately using branch qualifiers.
```

Comment:
Nit: since the term _branch qualifier_ is first introduced/defined at this point, I'd put it in _italics_.